src/handbook/company/security/information-security.md

```diff
@@ -0,0 +1,195 @@
+---
+navTitle: Information Security Policy and Acceptable Use Policy
+---
+
+# Information Security Policy and Acceptable Use Policy
+
+| Policy owner   | Effective date |
+| -------------- | -------------- |
+| @ZJvandeWeg    | 2023-05-01     |
+
+
+## Overview
+
+This Information Security Policy is intended to protect FlowForge’s employees,
+partners and the company from illegal or damaging actions by individuals, either
+knowingly or unknowingly.
+
+Internet-related systems, including but not limited to computer equipment,
+software, operating systems, storage media, network accounts providing electronic
+mail, web browsing, and file transfers, are the property of FlowForge.
+
+These systems are to be used for business purposes in serving the interests of
+the company, and of our clients and customers in the course of normal operations.
+
+Effective security is a team effort involving the participation and support of
+every FlowForge employee or contractor who deals with information and/or
+information systems. It is the responsibility of every team member to read and
+understand this policy, and to conduct their activities accordingly.
+
+## Purpose 
+
+The purpose of this policy is to communicate our information security policies
+and outline the acceptable use and protection of FlowForge’s information and
+assets. These rules are in place to protect customers, employees, and FlowForge.
+Inappropriate use exposes FlowForge to risks including virus attacks, compromise
+of network systems and services, financial and reputational risk, and legal and
+compliance issues.
+
+The FlowForge "Information Security Policy" is comprised of this policy and all
+FlowForge policies referenced and/or linked within this document.
+
+## Scope
+
+This policy applies to the use of information, electronic and computing devices,
+and network resources to conduct FlowForge business or interact with internal
+networks and business systems, whether owned or leased by FlowForge, the employee,
+or a third party. All employees, contractors, consultants, temporary, and other
+workers at FlowForge and its subsidiaries are responsible for exercising good
+judgment regarding appropriate use of information, electronic devices, and
+network resources in accordance with FlowForge policies and standards, and local
+laws and regulations.
+
+This policy applies to employees, contractors, consultants, temporaries, and
+other workers at FlowForge, including all personnel affiliated with third
+parties. This policy applies to all FlowForge-controlled company and customer
+data as well as all equipment, systems, networks and software owned or leased by
+FlowForge.
+
+## Security Incident Reporting
+
+All users are required to report known or suspected security events or incidents,
```

Comment:
If we state our policy in this doc, let's keep the handbook DRY and link here from `handbook/development/security.md`?

Reply:
I considered that. The doc you refer to is more focused on product vulnerability disclosure from outside the company - rather than internal reporting of any security incidents. SO whilst it reuses the same contact email, I considered it a different policy.

Reply:
I've added a link to the product disclosure policy to specifically cover product vulnerabilities, rather than any company-related security issues.